### PR TITLE
XW-2586 | styling PI flex (smart) groups

### PR DESCRIPTION
**Pull request by latata**
https://wikia-inc.atlassian.net/browse/XW-2586

### CODE REVIEW

File: extensions/wikia/PortableInfobox/services/PortableInfoboxRenderService.class.php [hunk 1]
```diff
@@ -263,15 +263,25 @@ private function createSmartGroups( $groupData, $rowCapacity ) {
...
-		return array_map( function ( $item ) use ( $capacity ) {
+	private function applyRowItemsStyles( $rowItems, $capacity, $isLastRow = false ) {
+		$items = array_map( function ( $item, $index ) use ( $capacity, $isLastRow ) {
 			$width = round( ( $item['data']['span'] / $capacity ) * 100 );
 			$item['data']['cssClasses'] = 'pi-smart-data';
+			if ( $isLastRow ) {
+				$item['data']['cssClasses'] .= ' last-row';
+			}
+			if ( $index % 2 === 1 ) {
+				$item['data']['cssClasses'] .= ' even-in-row';
```
Comment (rybmat) [inline extensions/wikia/PortableInfobox/services/PortableInfoboxRenderService.class.php]: what about .pi-smart-data__even-in-row? other classes also should be prefixed

File: extensions/wikia/PortableInfobox/styles/PortableInfobox.scss [hunk 2]
```diff
@@ -79,9 +79,31 @@
...
 
 .pi-smart-data {
+	border-left-style: solid;
+	border-left-width: 1px;
 	display: block;
 	flex-direction: column;
 	text-align: center;
+
+	.pi-data-value {
+		padding: 5px 10px 0;
```
Comment (rybmat) [inline extensions/wikia/PortableInfobox/styles/PortableInfobox.scss]: are these values used only here? Maybe it is worth to extract them to variables?

File: extensions/wikia/PortableInfobox/services/PortableInfoboxRenderService.class.php (hunk 1 above)
Comment (rybmat) [reply extensions/wikia/PortableInfobox/services/PortableInfoboxRenderService.class.php]: or per @adam suggestion just pi-even-in-row as we do not use BEM in portable infoboxes

Comment (idradm) [reply extensions/wikia/PortableInfobox/services/PortableInfoboxRenderService.class.php]: Let's keep those names as short as possible, but please keep the `pi-` prefix. As for the BEM notation, it is not used in PIs anywhere, so let's be consistent with that, and remember that users use those class names as well.

File: extensions/wikia/PortableInfobox/styles/PortableInfobox.scss (hunk 2 above)
Comment (idradm) [reply extensions/wikia/PortableInfobox/styles/PortableInfobox.scss]: 👍

**Review by idradm — CHANGES_REQUESTED** [review]

**Review by rybmat — APPROVED** [review]

**Review by idradm — APPROVED** [review]

**Pull request merged**